modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -93,7 +101,13 @@ class NDB_Form_user_accounts extends NDB_Form
             $send = $values['SendEmail'];
 	        unset($values['SendEmail']);
 	    }
-
+        
+        //store the supervisors emails
+        if (!empty($values['supervisorEmail'])) {
+            $supervisorEmails = $values['supervisorEmail'];
+            unset($values['supervisorEmail']);
+        }
```

Comment:
Shouldn't the supervisor emailing be based on the project config, not posted based on the user? I'm not sure what the point is if the editor can toggle whether or not they get emailed when changing the permissions.